bouncer/src/repo/lib/repo_utils.h

```diff
@@ -30,16 +30,30 @@ namespace repo {
 			return boost::filesystem::exists(inputPath) && boost::filesystem::is_directory(inputPath);
 		}
 
-		static std::string getExtension(const boost::filesystem::path &inputPath) {
+		static std::string getExtension(const boost::filesystem::path &inputPath) 
+		{
 			return inputPath.extension().string();
 		}
 
-		static void toLower(std::string &str) {
+		static void toLower(std::string &str) 
+		{
 			std::transform(str.begin(), str.end(), str.begin(), [](unsigned char c) { return std::tolower(c); });
 		}
 
-		static void toUpper(std::string &str) {
+		static void toUpper(std::string &str) 
+		{
 			std::transform(str.begin(), str.end(), str.begin(), [](unsigned char c) { return std::toupper(c); });
 		}
+
+		static std::string getEnvString(char const * const & envVarName)
+		{
+			std::string envVarStr;
+			char* envVarStsPtr = getenv(envVarName);
```

Comment:
`envVarStsPtr` is a bit of a weird naming, i can't actually figure out what the `Sts` part means.

given the context of the function, we could just call it `value`